test/gui/shared/steps/steps.py

```diff
@@ -402,18 +402,19 @@ def step(context, filename):
     waitForObject(names.settings_OCC_SettingsDialog)
     waitForObjectExists({
             "column": 1,
-            "container": names.oCC_IssuesWidget_treeWidget_QTreeWidget,
+            "container": names.oCC_IssuesWidget_tableView_QTableView,
             "text": RegularExpression(buildConflictedRegex(filename)),
             "type": "QModelIndex"
     })
+    
+# {"column": 1, "container": oCC_IssuesWidget_tableView_QTableView, "row": 0, "type": "QModelIndex"}
 
-
-@When('the user selects the unsynced files tab with |integer| unsynced files')
-def step(context, number):
+@When('the user selects the unsynced files tab')
+def step(context):
     # TODO: find some way to dynamically select the tab name
     # It might take some time for all files to sync except the expected number of unsynced files
     snooze(10)
-    clickTab(waitForObject(names.stack_QTabWidget), "Not Synced ({})".format(number))
```

Comment:
Maybe I should rather display the number again.
@michaelstingl what do you think?

Reply:
> Maybe I should rather display the number again.

What number? The counter in the tab-head? Yeah, good idea.

Reply:
@TheOneRing @michaelstingl For now I think we'll have to merge this PR as it is and when you change the UI again let me. As I have to modify these tests so that they can work with new UI changes.

Reply:
I think one purpose of those tests is to detect regressions.
I'd say the missing counter is a regression.